applications/inventory-app/src/assets/styles/styles.css

```
@@ -0,0 +1,68 @@
+/* Top Level */
+
+.draggable-card {
+    background-color: rgba(39,39,39, 1.0);
```

```suggestion
    background-color: rgba(39, 39, 39, 1.0);
```
And similarly, elsewhere.